projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts

```diff
@@ -47,7 +47,7 @@ import { WaterfallChartService } from './waterfall-chart.service';
           [data]="this.segments"
           class="chart"
           [selection]="this.selectedNode"
-          (selectionChange)="this.onSelection($event)"
+          (selectionChange)="this.onSelection($event ? [$event] : [])"
```

Comment:
selectionChange already emits an array of rows right? 
```
  public readonly selectionsChange: EventEmitter<StatefulTableRow[]> = new EventEmitter<StatefulTableRow[]>();
```

Reply:
It does for table but not for sequence chart.

Reply:
oh this part was not in table.